CYTFM_064B0S2_4343W: Fails to complete post build operation

<!--

   ************************************** WARNING **************************************

   The ciarcom bot parses this header automatically. Any deviation from the 
   template may cause the bot to automatically correct this header or may result in a 
   warning message, requesting updates.

   Please ensure all sections of the template below are filled in and no changes 
   are made to the template format. Only bugs should be raised here as issues. 
   Questions or enhancements should instead be raised on our forums:
   https://forums.mbed.com/ .

   *************************************************************************************

-->

### Description of defect
The post build operation does not complete.
<!--
    Add detailed description of what you are reporting.
    Good example: https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html
-->


#### Target(s) affected by this defect ?
`CYTFM_064B0S2_4343W`

#### Toolchain(s) (name and version) displaying this defect ?
ARM, GCC_ARM

#### What version of Mbed-os are you using (tag or sha) ?
<!--
    For a released version please provide the release tag (this can be found as per the instructions below)

    mbed-os version can be found in /platform/mbed_version.h. The tag can be reconstructed as follows:
    mbed-os-MBED_MAJOR_VERSION.MBED_MINOR_VERSION.MBED_PATCH_VERSION
 
    Master branch is indicated by 'mbed-os-99.99.99
    
    For an issue found on Master please provide the sha being used.
-->
`mbed-os-6.8.0`

#### What version(s) of tools are you using. List all that apply (E.g. mbed-cli)
```
$ mbed-tools --version
7.1.0
```

#### How is this defect reproduced ? 
Attempt to build `mbed-os-example-blinky` run the following command:
`rm -rf cmake_build/ && mbedtools compile -t gcc_ARM -m CYTFM_064B0S2_4343W`

Outcome:
```
...
Total Static RAM memory (data + bss): 11236(+11236) bytes
Total Flash memory (text + data): 51732(+51732) bytes

[338/338] Generating mbed-os-example-blinky.bin
FAILED: mbed-os/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CYTFM_064B0S2_4343W/mbed-os-example-blinky.bin 
cd /path/to/mbed-os-example-blinky/cmake_build/CYTFM_064B0S2_4343W/develop/GCC_ARM/mbed-os/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CYTFM_064B0S2_4343W && /usr/local/Frameworks/Python.framework/Versions/3.9/bin/python3.9 /path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/scripts/PSOC6.py sign --build-dir /path/to/mbed-os-example-blinky/cmake_build/CYTFM_064B0S2_4343W/develop/GCC_ARM --m0hex-filename tfm_s.hex --target-name CYTFM_064B0S2_4343W --policy-file-name policy_multi_CM0_CM4_tfm.json --boot-scheme multi_image --cm0-img-id 1 --cm4-img-id 16 --elf /path/to/mbed-os-example-blinky/cmake_build/CYTFM_064B0S2_4343W/develop/GCC_ARM/mbed-os-example-blinky.elf --m4hex /path/to/mbed-os-example-blinky/cmake_build/CYTFM_064B0S2_4343W/develop/GCC_ARM/mbed-os-example-blinky.hex --m0hex /path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CYTFM_064B0S2_4343W/COMPONENT_TFM_S_FW/tfm_s.hex
2021-02-22 13:43:16,055 : C : ERROR : Cannot find image with id 1 and type 'BOOT' in the policy file. Check the log for details
2021-02-22 13:43:16,055 : C : ERROR : Image with ID 1 not found in '/path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CYTFM_064B0S2_4343W/policy/policy_multi_CM0_CM4_tfm.json'. Check the log for details
Traceback (most recent call last):
  File "/path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/scripts/PSOC6.py", line 393, in <module>
    args.func(args)
  File "/path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/scripts/PSOC6.py", line 316, in sign_action
    sign_hex(
  File "/path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/scripts/PSOC6.py", line 229, in sign_hex
    sign_application(notification.debug, tools, m0hex, image_id=cm0_img_id)
  File "/path/to/mbed-os-example-blinky/mbed-os/targets/TARGET_Cypress/scripts/PSOC6.py", line 254, in sign_application
    + str(image_id) + " is " + hex(address) + ", " + hex(size))
TypeError: 'NoneType' object cannot be interpreted as an integer
ninja: build stopped: subcommand failed.
ERROR: CMake invocation failed!

More information may be available by using the command line option '-v'.
```

--- a/targets/TARGET_Cypress/scripts/PSOC6.py
+++ b/targets/TARGET_Cypress/scripts/PSOC6.py
@@ -369,10 +369,10 @@ def parse_args():
         "--boot-scheme", help="the boot scheme."
     )
     sign_subcommand.add_argument(
-        "--cm0-img-id", help="the Cortex-M0 image ID."
+        "--cm0-img-id", type=int, help="the Cortex-M0 image ID."
     )
     sign_subcommand.add_argument(
-        "--cm4-img-id", help="the Cortex-M4 image ID."
+        "--cm4-img-id", type=int, help="the Cortex-M4 image ID."
     )
     sign_subcommand.add_argument(
         "--elf", required=True, help="the application ELF file."

